doc/nrf/device_guides/working_with_nrf/nrf70/developing/fw_patches_ext_flash.rst

```diff
@@ -0,0 +1,133 @@
+.. _ug_nrf70_developing_fw_patch_ext_flash:
+
+Firmware patches in the external flash
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Overview
+********
+
+The nRF70 Series firmware patch is memory intensive and is by default part of the internal flash.
+In order to save flash memory for resource constrained hosts, the nRF70 Series offers a feature to move firmware patches to the external flash, if the external flash is available.
+
+Pre-requisites
+--------------
+
+The following conditions must be met to use this feature:
+
+- The external flash must be available and configured in the device tree.
+- The external flash must be large enough to store the firmware patches, maximum size is 128 KB.
+
+Constraints
+-----------
+
+The following constraints apply to this feature:
+
+- Access to the external flash is only supported via the QSPI interface for now.
+- In nRF SoC's XIP feature is only available via the QSPI interface only.
+
+
+Supported platforms
+-------------------
+
+The following platforms are supported:
+
+- nRF5340 SoC or DK
+- nRF52840 SoC or DK
+
+.. note:: Due to constraints mentioned above, the nRF7002 SoC or DK is not supported as it needs access to external flash via the SPI interface.
```

Comment:
What do we mean about the nRF7002 SoC is not supported? This is a wifi feature, so I assume that it works for the nRF7002 SoC, but on an EK. Therefore it should probably just say "the nRF7002DK is not supported" rather than  mentioning the SoC itself?

Reply:
Also, we say "it needs access to external flash via the SPI interface". But the nRF7002DK is connected to the external flash via SPI, and to the nRF7002 via QSPI.
Do we mean this instead:
```suggestion
.. note:: Due to constraints mentioned above, the nRF7002 SoC or DK is not supported as it needs access to external flash via the QSPI interface.
```

Reply:
The constraint above says `Access to the external flash is only supported through the QSPI interface.`, so, I was referring `nrf7002 external flash access is via SPI, hence not supported`.